src/confcom/azext_confcom/template_util.py

```diff
@@ -458,6 +457,83 @@ def filter_non_pod_resources(resources: List[dict]) -> List[dict]:
     return [resource for resource in resources if resource and resource.get("kind") in important_resource_names]
 
 
+def process_env_vars_from_config(container) -> List[Dict[str, str]]:
+    env_vars = []
+    # add in the env vars from the template
+    template_env_vars = case_insensitive_dict_get(
+        container, config.ACI_FIELD_TEMPLATE_ENVS
+    ) or []
+    for env_var in template_env_vars:
+        name = case_insensitive_dict_get(env_var, "name")
+        secure_value = case_insensitive_dict_get(env_var, "secureValue")
+        is_secure = bool(secure_value)
+        value = case_insensitive_dict_get(env_var, "value") or secure_value
+
+        if not name and not is_secure:
+            eprint(
+                f"Environment variable with value: {value} is missing a name"
+            )
+        elif not name and is_secure:
+            eprint(
+                "Environment variable with secure value is missing a name"
+            )
+        elif not value:
+            eprint(f'Environment variable {name} does not have a value. Please check the template file.')
+
+        env_vars.append({
+            config.ACI_FIELD_CONTAINERS_ENVS_NAME: name,
+            config.ACI_FIELD_CONTAINERS_ENVS_VALUE: value,
+            config.ACI_FIELD_CONTAINERS_ENVS_STRATEGY:
+                "re2" if case_insensitive_dict_get(env_var, "regex") else "string",
+        })
+
+    return env_vars
+
+
+def process_fragment_imports(rego_imports) -> None:
+    for rego_import in rego_imports:
+        feed = case_insensitive_dict_get(
+            rego_import, config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_FEED
+        )
+        if not isinstance(feed, str):
+            eprint(
+                f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
+                + f'["{config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_FEED}"] '
+                + "can only be a string value."
+            )
+
+        iss = case_insensitive_dict_get(
+            rego_import, config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_ISSUER
+        )
+        if not isinstance(iss, str):
+            eprint(
+                f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
+                + f'["{config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_ISSUER}"] '
+                + "can only be a string value."
+            )
+
+        minimum_svn = case_insensitive_dict_get(
+            rego_import, config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_MINIMUM_SVN
+        )
+
+        if not minimum_svn or not isinstance(minimum_svn, str) or not minimum_svn.isdigit():
```

Comment:
if it's a string shouldn't we check that we can cast it to a digit?

Reply:
`isdigit()` is a string function that does that for us